Lib/test/test_grammar.py

```diff
@@ -828,6 +834,8 @@ def g(): f((yield 1))
         def g(): f((yield 1), 1)
         def g(): f((yield from ()))
         def g(): f((yield from ()), 1)
+        # Do not require parenthesis for tuple unpacking
+        def g(): rest = 4, 5, 6; yield 1, 2, 3, *rest
```

Comment:
I would add another test here to validate that this works, e.g. `self.assertEqual(list(g()), [(1, 2, 3, 4, 5, 6)])`.

Reply:
I was tempted to do `self.assertEqual(*g(), (1, 2, 3, 4, 5, 6))`, but I think we're testing enough unpacking